libraries/rush-lib/src/api/LastInstallFlag.ts

```diff
@@ -80,6 +104,29 @@ export class LastInstallFlag {
                 `New Path: ${newStorePath}`
             );
           }
+
+          // check ignoreScripts
+          if (newState.ignoreScripts !== oldState.ignoreScripts) {
+            return false;
+          } else {
+            // full install
+            if (newState.selectedProjectNames === oldState.selectedProjectNames) {
```

Comment:
For clarity, `!newState.selectedProjectNames && !oldState.selectedProjectNames`; since if the value is an array they will always be inequal.